content/en/docs/event-sources/drivers.md

```diff
@@ -1,57 +1,62 @@
 ---
-title: Falco Drivers
+title: Syscall Events
```

Comment:
For all's worth, I think a good name could be "**Kernel Events**" here. We don't just push syscalls, but also things like container metadata, events from various tracepoints etc. Plus, the term "Kernel" also gathers in its umbrella concepts such as gVisor, pdig, or trace files, which don't really install a driver but still deal with events coming from the kernel. What do you think? @Andreagit97 @vjjmiras

Reply:
What I hear is that we need to distinguish between **Events** and **Event sources**. Currently we use use different sources (kernel module, eBPF, pdig, gVisor) to obtain the same kind of events (syscalls). 
Hence, **Kernel Events** would be partly incorrect, for not being a source, but partly correct, for grouping these different sources (since having separate sections might not be worth it).

Reply:
That's a good point, you bought me I will change it @vjjmiras @jasondellaluce

Reply:
In the falco terminology, a **source** is a stream of events with a certain structure. For example, the `syscall` source is composed of events collected from the kernel, and can be provided by the kmod, eBPF, gVisor, and userspace instrumentation. Another example is the `k8s_audit` source, which is composed of Kubernetes Audit Events. You could say that an event source is characterized by the events that get streamed through it.

As such, kmod/eBPF/gVisor are not sources, but just a way to instrument the kernel to obtain events.

Reply:
In the end `Kernel Events` seems reasonable for this context